lib/libzfs/libzfs_diff.c

```diff
@@ -344,12 +358,11 @@ describe_free(FILE *fp, differ_info_t *di, uint64_t object, char *namebuf,
 {
 	struct zfs_stat sb;
 
-	if (get_stats_for_obj(di, di->fromsnap, object, namebuf,
-	    maxlen, &sb) != 0) {
-		return (-1);
-	}
+	get_stats_for_obj(di, di->fromsnap, object, namebuf,
```

Comment:
Nit - Should we cast return value to (void) as that is the convention in the file?

Reply:
I've got no objection. Done.

Reply:
Great. Thanks!